server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java

```diff
@@ -332,4 +332,10 @@ public void accessTokenResponse() throws Throwable {
     public void assertContentDisposition(String name) throws IOException {
         assertThat(response.getHeaders("Content-Disposition")).extracting(Header::toString).containsExactly("Content-Disposition: attachment; filename=\"" + name + "\"");
     }
+
+    @Then("^the non ASCII attachment is named \"([^\"]*)\"$")
```

Comment:
it should not be a new sentence, the existing one is supposed to work, just add a if that checks asciiness of input